### PR TITLE
Changed the method of the route to POST

### PR DESCRIPTION
**Pull request by HarisShahidHameed**
Trello-link : https://trello.com/c/AJobiCqi/83-forget-password

### CODE REVIEW

File: server/routes/user.js [hunk 1]
```diff
@@ -176,7 +176,7 @@ router.patch('/user/:uuid/update/about', (req,res) => {
 })
 
 
-router.get('/user/emailExist', (req,res) => {
+router.post('/user/emailExist', (req,res) => {
```
Comment (abdulhaqshah) [inline server/routes/user.js]: its not a post its a get command and send an email in the param's not in body

**Review by abdulhaqshah — CHANGES_REQUESTED** [review]

File: server/routes/user.js (hunk 1 above)
Comment (HarisShahidHameed) [reply server/routes/user.js]: Done

File: server/libs/userlibs.js [hunk 2]
```diff
@@ -519,7 +519,8 @@ const checkByEmail = function(query) {
             if (user) {
                 resolve({
                     status : statusCodes.successful,
-                    message : `User ${messages.exist}`
+                    message : `User ${messages.exist}`,
+                    data : user
```
Comment (abdulhaqshah) [inline server/libs/userlibs.js]: why are you sending user data??

**Review by abdulhaqshah — CHANGES_REQUESTED** [review]

File: server/libs/userlibs.js (hunk 2 above)
Comment (HarisShahidHameed) [reply server/libs/userlibs.js]: Removed

**Review by abdulhaqshah — APPROVED** [review]

**Pull request merged**